docpages/example_programs/music_and_audio/soundboard.md

```diff
@@ -53,8 +49,8 @@ int main(int argc, char const *argv[])
 			
 			/* Tell the user we joined their channel. */
 			event.reply("Joined your channel!");
-		} else if (event.command.get_command_name() == "robot") {
```

Comment:
This is inconsistent with other changes you've made

Reply:
I absolutely did break every `} else {` I spotted, so not sure where you get that?

Reply:
Oh that appears to be correct yes
I think I got confused because in the majority of the code in DPP it's written as } else {

Reply:
Oh is it? It was definitely mixed in the examples, so I just decided to unify them this way, not a mention on it in the Style Guide. If we want them the other way, that can be done ofc.